<launch>
  <arg name="joy" default="/joy_teleop/joy" doc="Name of the topic the joystick is publishing on" />
  <arg name="open_value" default="0.0" doc="Position of the fingers (between [0,1]) to consider as 'open'" />
  <arg name="close_value" default="1.0" doc="Position of the fingers (between [0,1]) to consider as 'close'" />

  <node pkg="ur5-cartesian-control" name="joy_control" type="joystick_control.py" output="screen">
    <param name="joy_topic" value="$(arg joy)" />
    <param name="gripper_open_value" value="$(arg open_value)" />
    <param name="gripper_close_value" value="$(arg close_value)" />
  </node>
</launch>
